packages/block-editor/src/components/grid/style.scss

```diff
@@ -48,17 +44,26 @@
 
 	}
 
+	.block-editor-grid-visualizer__drop-zone {
+		opacity: 0;
+	}
+
 	&.is-highlighted {
 		.block-editor-inserter,
 		.block-editor-grid-visualizer__drop-zone {
-			background: var(--wp-admin-theme-color);
+			background: rgba(var(--wp-admin-theme-color--rgb), 0.25);
+			border: 1px solid var(--wp-admin-theme-color);
 		}
 	}
 
 	&:hover .block-editor-grid-visualizer__appender,
 	.block-editor-grid-visualizer__appender:focus {
 		opacity: 1;
-		background-color: color-mix(in srgb, currentColor  20%, #0000);
+		box-shadow: none;
+		background-color: rgba(var(--wp-admin-theme-color--rgb), 0.05);
+		border: 1px dashed;
+		border-radius: 2px;
```

Comment:
I think we have a `$border-radius` var for this.

Reply:
Oh yep, that was lazy. Now updated.